ddp-workspace/projects/ddp-rgp/src/styles/_activity.scss

```diff
@@ -213,14 +213,14 @@
   margin-bottom: 10px;
   padding: 0.4375em 0;
 
-  border-bottom: 1px solid rgba(0, 0, 0, 0.42);
+  // border-bottom: 1px solid rgba(0, 0, 0, 0.42);
```

Comment:
Not sure what changed recently in Angular side, but for RGP we have "double underline", so removing this bottom border.

Before:
![rgp_bug_underline_before](https://user-images.githubusercontent.com/5713833/116941827-77099a80-ac3e-11eb-95f7-7cd97be0b526.png)

After:
![rgp_bug_underline_after](https://user-images.githubusercontent.com/5713833/116941851-7ec93f00-ac3e-11eb-8b18-c7c7a750b394.png)